workers/generic-worker/runtime_simple.go

```diff
@@ -0,0 +1,14 @@
+//go:build simple
+
+package main
+
+import (
+	"github.com/taskcluster/taskcluster/v58/workers/generic-worker/process"
+	"github.com/taskcluster/taskcluster/v58/workers/generic-worker/runtime"
+)
+
+// This is executed as the task user to ensure that the
+// generic-worker binary is readable/executable.
```

Comment:
Something like:

gwVersion executes `generic-worker --version` as the task user. This is useful to test that that the task user has permission to run generic-worker subcommands. It does not return the version as a result - the result is already available as a global variable in the main package. A non-nil error is returned if the command could not be executed.

We could call the function something like testExecutingGWSubcommandAsTaskUser to make it clearer that it isn't a function for getting the gw version.